cmd/transactioncmd/transaction_commit.go

```diff
@@ -24,9 +24,6 @@ func newTransactionCommitCmd() *cobra.Command {
 	}
 
 	cmd.Flags().StringVar(&inputTxPath, inputTxPathFlag, "", "Path to the transaction signed by all signatories")
-	if err := cmd.MarkFlagRequired(inputTxPathFlag); err != nil {
```

Comment:
You removed the requirement, but you didn't add a prompt. Now I think it will error if you don't provide the flag

Reply:
The empty check + prompt was already there in the code

Reply:
ahh, it didn't show up in the diff